torch/_inductor/overrides.py

```diff
@@ -133,6 +191,41 @@ def fuse_conv_unary_eval(conv: nn.Module, unary: nn.Module):
     )
 
 
+def fuse_conv_binary_eval(conv: nn.Module, binary_op_name: str):
+    assert not (conv.training), "Fusion only for eval!"
+    return ConvBinary2d(
+        conv,
+        binary_op_name,
+    )
+
+
+def check_node_kind(current_node, modules, node_kind):
+    if not isinstance(current_node, torch.fx.Node):
+        return False
+    if current_node.op != "call_module":
+        return False
+    if not isinstance(current_node.target, str):
+        return False
+    if current_node.target not in modules:
+        return False
+    if type(modules[current_node.target]) is not node_kind:
+        return False
+    return True
+
+
+def check_node_is_binary(node):
+    if (
+        (node.op == "call_function" and node.target in [torch.add, torch.sub])
+        or (node.op == "call_function" and node.target in [operator.add, operator.sub])
+        or (
+            node.op == "call_method"
+            and node.target in [torch.Tensor.add, torch.Tensor.sub]
+        )
```

Comment:
Simpler to just return the value of this boolean expression instead of an extra "if-else" here?